packages/react-native/ReactAndroid/src/main/java/com/facebook/react/bridge/DynamicFromArray.kt

```diff
@@ -0,0 +1,65 @@
+/*
+ * Copyright (c) Meta Platforms, Inc. and affiliates.
+ *
+ * This source code is licensed under the MIT license found in the
+ * LICENSE file in the root directory of this source tree.
+ */
+
+package com.facebook.react.bridge
+
+import androidx.core.util.Pools
+
+/** Implementation of Dynamic wrapping a ReadableArray. */
+internal class DynamicFromArray
+private constructor(private var array: ReadableArray? = null, private var index: Int = -1) :
```

Comment:
In the original Java file, the variables were not initialized in the constructor, could you double check that?